src/services/TrainingService.js

```diff
@@ -62,12 +63,8 @@ export default {
   saveAnswer (context, picked) {
     const question = this.questions[this.index]
     if (
-      picked !== '' &&
-      picked !== null &&
-      picked !== undefined &&
-      (this.idAnswerMap[question._id] === null ||
-        this.idAnswerMap[question._id] === '' ||
-        picked !== undefined)
+      _.isEmpty(this.idAnswerMap[question._id]) &&
+      !_.isEmpty(picked)
```

Comment:
A code comment or [explaining variable](http://wiki.c2.com/?IntroduceExplainingVariable) here would be helpful.

Reply:
Oh yeah good point!